### DISCUSSION TITLE
vue3-element-plus可以使用的formily2设计器

### DISCUSSION THREAD

**Discussion by jackverse** [alibaba/formily#3102] [Q&A 2.x]
我想问一下技术栈是vue3加element-plus，如果想要调用设计器，有没有什么办法

**Comment by qq1037305420** [comment]
https://qq1037305420.github.io/element-plus/
源码：https://github.com/qq1037305420/element-plus

**Reply by jackverse** [reply]
谢谢您，我想咨询下您提供的设计器如果后期正式推出社区版的话（或者用您发给我的这个），我是在自己的项目里通过node_modules去引用，还是说有其他更好的办法，如果是放到node_module里的话，是直接将prototypes、renderer、setters、settings-form放到指定的位置就可以了吗，我朝这个方向测试，发现会出零零散散的问题，有点怀疑自己的研究方向是不是错了

**Reply by merryGirl** [reply]
大佬，本地运行设计器，拖拽失败了

**Reply by YaoHanhan** [reply]
我下载下来运行不了
